include/cantera/oneD/StFlow.h

```diff
@@ -325,25 +312,133 @@ class StFlow : public Domain1D
         return m_wdot(k,j);
     }
 
-    //! Write the net production rates at point `j` into array `m_wdot`
-    void getWdot(double* x, size_t j) {
-        setGas(x,j);
-        m_kin->getNetProductionRates(&m_wdot(0,j));
-    }
-
     //! Update the properties (thermo, transport, and diffusion flux).
     //! This function is called in eval after the points which need
     //! to be updated are defined.
     virtual void updateProperties(size_t jg, double* x, size_t jmin, size_t jmax);
 
-    //! Evaluate the residual function. This function is called in eval
-    //! after updateProperties is called.
-    virtual void evalResidual(double* x, double* rsd, int* diag,
+    /**
+     * Computes the radiative heat loss vector over points jmin to jmax and stores
+     * the data in the qdotRadiation variable.
+     *
+     * The simple radiation model used was established by Liu and Rogg
+     * @cite liu1991. This model considers the radiation of CO2 and H2O.
+     *
+     * This model uses the optically thin limit and the gray-gas approximation to
+     * simply calculate a volume specified heat flux out of the Planck absorption
+     * coefficients, the boundary emissivities and the temperature. Polynomial lines
+     * calculate the species Planck coefficients for H2O and CO2. The data for the
+     * lines are taken from the RADCAL program @cite RADCAL.
+     * The coefficients for the polynomials are taken from
+     * [TNF Workshop](https://tnfworkshop.org/radiation/) material.
+     */
+    void computeRadiation(double* x, size_t jmin, size_t jmax);
+
+    /**
+     * Evaluate the continuity equation residual.
+     *
+     * This function calculates the residual of the continuity equation
+     * @f[
+     *     \frac{d(\rho u)}{dz} + 2\rho V = 0
+     * @f]
+     *
+     * The continuity equation propagates information from right-to-left.
+     * The @f$ \rho u @f$ at point 0 is dependent on @f$ \rho u @f$ at point 1 but not
+     * on @f$ \dot{m} @f$ from the inlet. The default value for the continuity equation
+     * is zero velocity (@f$ u @f$) at the left and right boundary.
+     */
+    virtual void evalContinuity(double* x, double* rsd, int* diag,
+                                double rdt, size_t jmin, size_t jmax);
+
+    /**
+     * Evaluate the momentum equation residual.
+     *
+     * The function calculates the radial momentum equation defined as
+     * @f[
+     *    \rho \frac{dV}{dt} + \rho u \frac{dV}{dz} + \rho V^2 =
+     *    \frac{d(\mu \frac{dV}{dz})}{dz} - \lambda
```

Comment:
```suggestion
     *    \frac{d(\mu \frac{dV}{dz})}{dz} - \Lambda
```
In addition, I don't think there's a transient term here. For the Cantera Science version, please have a look here
https://github.com/Cantera/cantera-website/blob/73af4ef147ed7782df7da05ebf2a0a7a50ae0753/pages/science/flames.rst?plain=1#L54-L56

Reply:
Yes, you are correct.